src/main/java/com/ioet/bpm/skills/boundaries/SkillController.java

```diff
@@ -33,6 +34,12 @@ public SkillController(SkillRepository skillRepository) {
                 .orElseGet(() -> new ResponseEntity<>(HttpStatus.NOT_FOUND));
     }
 
+    @GetMapping("/search/{name}")
+    public ResponseEntity<List>getName(@PathVariable(value = "name") String skillName){
```

Comment:
The name should not be a `PathVariable`, it should be passed as a query parameter so we can do a search in this way:

http://IP:PORT/skills?name={provided_name_here}

Reply:
We do not use PathVariable, because it is not necessary to add the value of the variable to one of the arguments of the method?